src/main/scala/org/constellation/storage/SnapshotService.scala

```diff
@@ -406,7 +409,7 @@ class SnapshotService[F[_]: Concurrent](
   ): F[Snapshot] =
     storedSnapshot.get
       .map(_.snapshot.hash)
-      .map(hash => Snapshot(hash, hashesForNextSnapshot, SortedMap(publicReputation.toSeq: _*)))
+      .map(hash => snapshot.Snapshot(hash, hashesForNextSnapshot, SortedMap(publicReputation.toSeq: _*)))
```

Comment:
Isn't Snapshot imported above? Shouldn't it work without specifying the package?

Reply:
It was done automatically by IntelliJ